qt/widgets/common/test/FitScriptGeneratorViewTest.h

```diff
@@ -0,0 +1,205 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "MantidKernel/WarningSuppressions.h"
+#include "MantidQtWidgets/Common/FitScriptGeneratorDataTable.h"
+#include "MantidQtWidgets/Common/FitScriptGeneratorView.h"
+#include "MantidQtWidgets/Common/IFitScriptGeneratorPresenter.h"
+
+#include <cxxtest/TestSuite.h>
+#include <gmock/gmock.h>
+
+#include <memory>
+
+#include <QApplication>
+#include <QTableWidget>
+#include <QtTest>
+
+using namespace MantidQt::MantidWidgets;
+using namespace testing;
+
+GNU_DIAG_OFF_SUGGEST_OVERRIDE
```

Comment:
I noticed that the view had about 5 methods with logic, but I cannot see any obvious tests. Could we have some please?

Reply:
Good spot 👍